generators/app/templates/src/screens/Registration/index.tsx

```diff
@@ -14,34 +15,42 @@ import styles from './styles.module.scss';
 
 const PASSWORD_LENGTH = 8;
 
-const FIELDS = {firstName: 'firstName', lastName: 'lastName', email: 'email', password: 'password', confirmPassword: 'confirmPassword'};
-
-const requiredValidation = { required: i18next.t('Registration:requiredError') as string };
+const FIELDS = {
+  firstName: 'firstName',
+  lastName: 'lastName',
+  email: 'email',
+  password: 'password',
+  confirmPassword: 'confirmPassword'
+};
 
-const VALIDATIONS = {
-  firstName: requiredValidation,
-  lastName: requiredValidation,
-  email: {
-    pattern: {
-      value: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
-      message: i18next.t('Registration:emailFormatError') as string
+const getValidations = (t: TFunction) => {
+  const requiredValidation = { required: t('requiredError') };
+  return {
+    firstName: requiredValidation,
+    lastName: requiredValidation,
+    email: {
+      pattern: {
+        value: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+        message: t('emailFormatError') as string
+      },
+      ...requiredValidation
     },
-    ...requiredValidation
-  },
-  password: {
-    ...requiredValidation,
-    minLength: {
-      value: 8,
-      message: i18next.t('Registration:passwordLengthError', { amount: PASSWORD_LENGTH }) as string
-    }
-  },
-  confirmPassword: requiredValidation
+    password: {
+      ...requiredValidation,
+      minLength: {
+        value: 8,
+        message: t('passwordLengthError', { amount: PASSWORD_LENGTH }) as string
+      }
+    },
+    confirmPassword: requiredValidation
+  };
 };
 
-function RegistrationContainer() {
+function Registration() {
   const { register, handleSubmit, errors, watch } = useForm();
   const dispatch = useDispatch();
   const history = useHistory();
+  const { t } = useTranslation(['Registration', 'RecoverPassword']);
```

Comment:
Why do you need `RecoverPassword` here? I'd keep the screens independant.

Reply:
i kept the old translations but i can move those translations here

Reply:
I'd only keep `Registration`